lib/include/prjxray/xilinx/bitstream_reader.h

```diff
@@ -77,10 +85,61 @@ class BitstreamReader {
 
        private:
 	static std::array<uint8_t, 4> kSyncWord;
+	const std::vector<uint32_t> kWcfgCmd = {0x30008001, 0x1};
+	const std::vector<uint32_t> kNullCmd = {0x30008001, 0x0};
 
 	std::vector<uint32_t> words_;
 };
 
+// Extract FPGA configuration logic information
+template <typename ArchType>
+void BitstreamReader<ArchType>::ExtractFpgaConfigurationLogicData(
+    FILE* aux_fp) {
+	// Get the data before the first FDRI_WRITE command packet
+	const auto fpga_conf_end = std::search(
+	    words_.cbegin(), words_.cend(), kWcfgCmd.cbegin(), kWcfgCmd.cend());
+	fprintf(aux_fp, "FPGA configuration logic prefix: ");
```

Comment:
If you remove the space from the end of this string, and add it to the one below, you can remove the `fseek`'s in this method.

Reply:
Right, done.